test/mocks/statement.of.capital.mock.ts

```diff
@@ -1,12 +1,28 @@
-import { StatementOfCapital } from "private-api-sdk-node/dist/services/confirmation-statement";
+import {
+  ConfirmationStatementSubmission, SectionStatus,
+  StatementOfCapital
+} from "private-api-sdk-node/dist/services/confirmation-statement";
 
 export const mockStatementOfCapital: StatementOfCapital = {
   classOfShares: "Ordinary",
   currency: "GBP",
-  numberAllotted: "10",
-  aggregateNominalValue: "1",
-  prescribedParticulars: "This is a test",
-  totalNumberOfShares: "10",
-  totalAggregateNominalValue: "10",
-  totalAmountUnpaidForCurrency: "1"
+  numberAllotted: "100",
+  aggregateNominalValue: "0.01",
+  prescribedParticulars: "THE QUICK BROWN FOX",
+  totalNumberOfShares: "100",
+  totalAggregateNominalValue: "1",
+  totalAmountUnpaidForCurrency: "2"
+};
+
+export const mockConfirmationStatementSubmission: ConfirmationStatementSubmission = {
```

Comment:
there is a mockConfirmationStatementSubmission in confirmation.statement.submission.mock. Looks very similar, could you move this mock into the confirmation.statement.submission.mock.ts file with the other and possibly re-use / combine with the existing one?